### ISSUE TITLE
Request: Support for Noizio

### ISSUE THREAD

**Issue by amitav13** [kyleneideck/BackgroundMusic#118]
Though it has a small userbase, [Noizio](http://noiz.io/) support would be great.

**Comment by kyleneideck** [comment]
Good idea, but I think Noizio might be a bit tricky to add support for.

Do you know if it has an API? Or a free/trial version? I wasn't able to find either.

If it has an AppleScript API, you should be able to open Script Editor, go File > Open Dictionary and select it from the list of apps.

If it doesn't have an API, I think we'd have to use the accessibility API to send simulated keystrokes to it or something like that. It'd be hard to do that securely. On the other hand, people have asked about at least a few other music players that would require the same, so it might be worthwhile.

The only other way I can think of is to have the user close Noizio so Background Music could launch it as a child process and control it. I think we'd want to avoid that solution if at all possible, and I'm not sure it would work, but it might be more secure because Background Music wouldn't need a process authorized as a [trusted accessibility client](https://developer.apple.com/documentation/applicationservices/1460720-axisprocesstrusted?language=objc).
